crates/bevy_render/src/render_resource/uniform_buffer.rs

```diff
@@ -99,23 +99,23 @@ impl<T: ShaderType + WriteInto> UniformBuffer<T> {
         self.label.as_deref()
     }
 
-    /// Queues writing of data from system RAM to VRAM using the [`RenderDevice`](crate::renderer::RenderDevice)
-    /// and the provided [`RenderQueue`](crate::renderer::RenderQueue), if a GPU-side backing buffer already exists.
+    /// Queues writing of data from system RAM to VRAM using the [`GPUDevice`](crate::renderer::GpuDevice)
+    /// and the provided [`GPUQueue`](crate::renderer::GpuQueue), if a GPU-side backing buffer already exists.
```

Comment:
```suggestion
    /// Queues writing of data from system RAM to VRAM using the [`GpuDevice`](crate::renderer::GpuDevice)
    /// and the provided [`GpuQueue`](crate::renderer::GpuQueue), if a GPU-side backing buffer already exists.
```

Reply:
There's a couple of those in the PR, so I won't add a suggestion everywhere, but for consistency it's probably better to use the same capitalization as the struct.